clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2581,16 +2581,17 @@ def SYCLIntelUseStallEnableClustersAttrDocs : Documentation {
   let Category = DocCatFunction;
   let Heading = "intel::use_stall_enable_clusters";
   let Content = [{
-When applied to a lambda or function call operator (of a function object)
-on device, this requests, to the extent possible, that statically-scheduled
-clusters handle stalls using a stall-enable signal to freeze computation
-within the cluster. This attribute is ignored on the host.
+When applied to a lambda function, function definition, or function call
+operator (of a function object) on device or host, this requests, to the
+extent possible, that statically-scheduled clusters handle stalls using a
+stall-enable signal to freeze computation within the cluster. This attribute
+is not ignored on the host.
 
-If ``intel::use_stall_enable_clusters`` is applied to a function called from a device
-kernel, the attribute is ignored and it is not propagated to the kernel.
+If ``intel::use_stall_enable_clusters`` is applied to a function called from a
+device kernel, the attribute is ignored and it is not propagated to the kernel.
```

Comment:
I think this line is incorrect. The attribute is not ignored. It is still applied to the function. You can probably just remove this line. We don't need to mention the kernel at all anymore in my opinion, since this has become a 'normal' function attribute